### PR TITLE
Bug 1685988 - Deploy messages WNPanel messages for 85

### PR DESCRIPTION
**Pull request by piatra**

### CODE REVIEW

File: whats-new-panel.yaml [hunk 1]
```diff
@@ -104,3 +62,41 @@
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_SUPERCOOKIE_85
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_SUPERCOOKIE_85
+    published_date: 1614038400000
```
Comment (ncloudioj) [inline whats-new-panel.yaml:70]: This converts to "Feb 23 2021", shall we set it as "Jan 26 2021" instead?

File: whats-new-panel.yaml [hunk 2]
```diff
@@ -104,3 +62,41 @@
...
+      string_id: cfr-whatsnew-pip-cta
+  targeting: firefoxVersion >= 85
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_BOOKMARKING_85
+  template: whatsnew_panel_message
+  order: 2
+  content:
+    bucket_id: WHATS_NEW_BOOKMARKING_85
+    published_date: 1614038400000
```
Comment (ncloudioj) [inline whats-new-panel.yaml:89]: Ditto.

File: whats-new-panel.yaml (hunk 1 above)
Comment (piatra) [reply whats-new-panel.yaml:70]: Thanks for catching!

File: whats-new-panel.yaml [hunk 3]
```diff
@@ -104,3 +62,41 @@
...
+    title:
+      string_id: cfr-whatsnew-bookmarking-header
+    icon_url: data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgaGVpZ2h0PSIxNiI+PHBhdGggZmlsbD0iIzQ1YTFmZiIgZmlsbC1vcGFjaXR5PSJjb250ZXh0LWZpbGwtb3BhY2l0eSIgZD0iTTE1Ljg0NSA2LjA2NEExLjEgMS4xIDAgMCAwIDE1IDUuMzMxTDEwLjkxMSA0LjYgOC45ODUuNzM1YTEuMSAxLjEgMCAwIDAtMS45NjkgMEw1LjA4OSA0LjZsLTQuMDgxLjcyOWExLjEgMS4xIDAgMCAwLS42MTUgMS44MzRMMy4zMiAxMC4zMWwtLjYwOSA0LjM2YTEuMSAxLjEgMCAwIDAgMS42IDEuMTI3TDggMTMuODczbDMuNjkgMS45MjdhMS4xIDEuMSAwIDAgMCAxLjYtMS4xMjdsLS42MS00LjM2MyAyLjkyNi0zLjE0NmExLjEgMS4xIDAgMCAwIC4yMzktMS4xeiIvPjwvc3ZnPg==
+    icon_alt: Bookmark icon
+    body:
+      string_id: cfr-whatsnew-bookmarking-body
+    cta_url: https://support.mozilla.org/1/firefox/%VERSION%/%OS%/%LOCALE%/media-keyboard-control
```
Comment (ncloudioj) [inline whats-new-panel.yaml:96]: This URL seems incorrect, can you confirm?

**Review by ncloudioj — APPROVED** [review]
:shipit:

**Comment by piatra** [comment]
Saving this for reference later if needed: https://sql.telemetry.mozilla.org/dashboard/messaging-system-what-s-new-panel-in-firefox-85-

**Pull request merged**